streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetrics.java

```diff
@@ -113,6 +115,22 @@ public static Sensor processLatencySensor(final String threadId,
         );
     }
 
+    public static Sensor totalInputBufferBytesSensor(final String threadId,
+        final String taskId,
+        final StreamsMetricsImpl streamsMetrics) {
```

Comment:
style consistent nit:
```suggestion
    public static Sensor totalInputBufferBytesSensor(final String threadId,
                                                     final String taskId,
                                                     final StreamsMetricsImpl streamsMetrics) {
```

Reply:
addressed in 25c75dc